### PR TITLE
fix: authenticate with keel

### PR DESCRIPTION
**Pull request by chanoe**

### CODE REVIEW

File: cmd/keel/root.go [hunk 1]
```diff
@@ -77,10 +82,19 @@ var rootCmd = &cobra.Command{
...
+			tokenStore := oredis.NewRedisStore(&redis.Options{
+				Addr:     conf.SecurityConf.OAuth.Redis.Addr,
+				DB:       conf.SecurityConf.OAuth.Redis.DB,
+				Password: conf.SecurityConf.OAuth.Redis.Password,
+			})
+			tokenGenerator := generates.NewJWTAccessGenerate("", []byte(conf.SecurityConf.OAuth.AccessGenerate.SecurityKey), jwt.SigningMethodHS512)
+
 			// init service.
 			// proxy service.
+			OauthSrv := service.NewOauthService(tokenConf, tokenStore, tokenGenerator, nil)
```
Comment (OdysseusC) [inline cmd/keel/root.go]:
```suggestion
			oauthSrv := service.NewOauthService(tokenConf, tokenStore, tokenGenerator, nil)
```

**Review by OdysseusC — COMMENTED** [review]
variable names should be lowercase.

**Review by OdysseusC — APPROVED** [review]

**Review by uptutu — APPROVED** [review]

**Pull request merged**